backend/tests/integration/generatePreviewSpark.api.test.js

```
@@ -210,50 +186,137 @@ describe('/v1/generate-preview-spark API (Integration, Defensive)', () => {
   });
 
   it('should enforce type safety and output structure', async () => {
-    // TODO: Use expectTypeOf/assertType if available, check output shape
+    // Defensive: See test plan and defensive implementation docs
+    const response = await request(app)
+      .post('/v1/generate-preview-spark')
+      .send(validInput)
+      .set('Accept', 'application/json');
+    // Type assertions (if expectTypeOf is available)
+    if (typeof expectTypeOf === 'function') {
+      expectTypeOf(response.body.previewSpark).toMatchTypeOf({
```

ESLint

[error] 'expectTypeOf' is not defined.

(no-undef)